content/en/docs/concepts/storage/volumes.md

```diff
@@ -1293,7 +1293,7 @@ Learn how to
 
 #### CSI ephemeral volumes
 
-{{< feature-state for_k8s_version="v1.14" state="alpha" >}}
+{{< feature-state for_k8s_version="v1.15" state="alpha" >}}
```

Comment:
Does feature-gates.md also need to be updated?

Reply:
@msau42 fixed!